plugins/module_utils/robot.py

```diff
@@ -105,7 +113,7 @@ def fetch_url_json(module, url, method='GET', timeout=10, data=None, headers=Non
         content = info.pop('body', None)
 
     if not content:
-        if allow_empty_result and info.get('status') in (200, 204):
+        if allow_empty_result and info.get('status') in (200, 201, 204):
```

Comment:
Same here.

Reply:
Done in 2c357753432436dad8d07cec0d737718ad388234